native/src/main/java/io/ballerina/stdlib/http/api/HttpResource.java

```diff
@@ -203,6 +204,14 @@ public void setTransactionInfectable(boolean transactionInfectable) {
         this.transactionInfectable = transactionInfectable;
     }
 
+    public void setTreatNilableAsOptional(boolean treatNilableAsOptional) {
+        this.treatNilableAsOptional = treatNilableAsOptional;
+    }
+
+    public boolean getTreatNilableAsOptional() {
```

Comment:
```suggestion
    public boolean isTreatNilableAsOptional() {
```

Reply:
Resolved in [`ee94230`](https://github.com/ballerina-platform/module-ballerina-http/pull/653/commits/ee94230d58b6a47ddad67b84ac4bcab5227ee6eb)